hack/generate-manifest-flow-visibility.sh

```diff
@@ -106,10 +110,27 @@ BASE=../../base
 
 mkdir $MODE && cd $MODE
 touch kustomization.yml
-$KUSTOMIZE edit add base $BASE
 # ../../patches/$MODE may be empty so we use find and not simply cp
 find ../../patches/$MODE -name \*.yml -exec cp {} . \;
 
+if [ "$MODE" == "e2e" ]; then
+    mkdir -p base/provisioning/datasources
+    cp $KUSTOMIZATION_DIR/base/clickhouse.yml base/clickhouse.yml
+    cp $KUSTOMIZATION_DIR/base/kustomization-e2e.yml base/kustomization.yml
+    cp $KUSTOMIZATION_DIR/base/kustomize-config.yml base/kustomize-config.yml
+    cp $KUSTOMIZATION_DIR/base/provisioning/datasources/create_table.sh base/provisioning/datasources/create_table.sh
+    cp $KUSTOMIZATION_DIR/../clickhouse-operator-install-bundle.yml clickhouse-operator-install-bundle.yml
+
+    $KUSTOMIZE edit add base base
+    $KUSTOMIZE edit add base clickhouse-operator-install-bundle.yml
+    $KUSTOMIZE edit add patch --path imagePullPolicyOperator.yml
+
+    $KUSTOMIZE edit set image flow-visibility-clickhouse-monitor=projects.registry.vmware.com/antrea/flow-visibility-clickhouse-monitor:latest
+    $KUSTOMIZE edit add patch --path imagePullPolicyClickhouse.yml --group clickhouse.altinity.com --version v1 --kind ClickHouseInstallation --name clickhouse
+else
+    $KUSTOMIZE edit add base $BASE
+fi
```

Comment:
I don't understand why we need so much customization compared to dev mode. Maybe it indicates that something should be improved?

Have you given any thought to what this will be like when the visibility solution is moved to the theia repo? Maybe you intend to improve things at that time?

Reply:
essentially what we're doing here is:
- re-create a kustomize folder to use kustomization-e2e.yml as base `kustomization.yml`, which only includes clickhouse db but not other grafana related manifest components
- since we don't want to dup other shared files (patches / configs / db init script) in the flow-visibility dir, we're copying them from the base dir instead of building within that dir. This make it look like having a lot of customization where half of the steps are just copying existing files.
- also includes `clickhouse-operator-install-bundle.yml` in one yaml file for easier manifest copying and deploying with test code. For release yaml it's a separate file.

Once FV manifest is moved out of Antrea we'll have a simplified folder structure dedicated for e2e test. At that time all these copying is not necessary.